### ISSUE TITLE
[Components and pattern standards] Your content or feature doesn't use standard global navigation components or is missing a standard navigation component.  (04.02.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#54299]
#### VFS team name
Sitewide Facilities

#### VFS product name
VAMC

#### VFS feature name
VAMC Lovell Federal Health care

#### Point of Contact/Reviewers
Sara Sterkenburg - @sterkenburgsara - IA

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Your content or feature doesn't use standard global navigation components or is missing a standard navigation component. 

### Issue Details
All breadcrumbs (except for "events" page) are missing the initial segment "home" that would route users back to VA.gov home.

### Link, screenshot or steps to recreate
https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/52672_Sara_040201.png

### VA.gov Experience Standard
[Category Number 04, Issue Number 02](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
I synced with Mikki Northuis on this one, and she let me know that your previous testing had shown that users clicked home intending to reach the landing page for the Lovell site, not the larger Va.gov home. There is a fix/solution in place for this though, which is to use the breadcrumb segment "Va.gov home" instead of just "home." Please add this breadcrumb segment to all affected pages within your product.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by davidmpickett** [comment]
Removing the Lovell tag as this is an issue for all VAMCs

**Comment by mmiddaugh** [comment]
See also [Discovery: Intended breadcrumb behavior (VAMC)](https://app.zenhub.com/workspaces/facilities-speedrun-643dd505bb64770033617caa/issues/gh/department-of-veterans-affairs/va.gov-team/13571)

**Comment by jilladams** [comment]
This ticket showed up In Progress when we transitioned boards, so I took a look to see if the bigger picture breadcrumb / component migration effort to add VA.gov home to all breadcrumbs has fixed it. However, it doesn't include any flavor of Home on Lovell: 

![Screenshot 2024-06-27 at 4 20 33 PM](https://github.com/department-of-veterans-affairs/va.gov-team/assets/85581471/025b00df-8f87-4681-a1af-da4b95207351)

Not urgent, just noting it while I'm here. I'll chat with Laura about why it's in progress sometime when things are less busy.

**Comment by laflannery** [comment]
With the upgrade to v3 the breadcrumb component this is now resolved: 
- [Lovell Federal VA system page](https://www.va.gov/lovell-federal-health-care-va/)
![image](https://github.com/user-attachments/assets/8ad462ec-31ab-41f7-88b9-9a4be392d26f)

- [Lovell Federal TRICARE system page](https://www.va.gov/lovell-federal-health-care-tricare/)
![image](https://github.com/user-attachments/assets/930c9d2d-b9c7-4943-a3e1-2c2d7c412177)
